app/src/main/java/com/hedvig/app/feature/addressautocompletion/activityresult/FetchDanishAddressAutoCompleteContractHandler.kt

```diff
@@ -0,0 +1,39 @@
+package com.hedvig.app.feature.addressautocompletion.activityresult
+
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.ActivityResultRegistry
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import com.hedvig.app.feature.addressautocompletion.model.DanishAddress
+
+/**
+ * Must remember to register as a lifecycle observer in the `onCreate` method of the activity.
+ * More information: https://developer.android.com/training/basics/intents/result#separate
+ */
+class FetchDanishAddressAutoCompleteContractHandler(
```

Comment:
This along with the FetchDanishAddressContract below contain all the Activity result API classes to make it work